examples/async_transcribe_local_media_file.js

```diff
@@ -26,7 +26,8 @@ const token = require('./config/config.json').access_token;
             ]
         }],
         filter_profanity: false,
-        remove_disfluencies: false
+        remove_disfluencies: false,
+        delete_after_seconds: 2592000   // 30 days in seconds
```

Comment:
```suggestion
        delete_after_seconds: 2592000 // 30 days in seconds
```
Very minor - I think too many spaces 😛